public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width initial-scale=1.0" />
     <title>TestBrew</title>
-    <script defer src="bundle.js"></script>
+    <script defer src="/bundle.js"></script>

```

Comment:
what does this change do (usually, I don't include the forward slash in this script element)?

Reply:
If I make a component file in a deeper path, the component file will not be found. I'll show you later.